app/src/components/collectionLanes/collectionLanes.tsx

```diff
@@ -8,28 +8,31 @@ import {
   Spacer,
   Link,
 } from "@nypl/design-system-react-components";
-import styles from "./Swimlanes.module.css";
+import styles from "./Collectionlanes.module.css";
 import { DC_URL } from "../../config/constants";
 import useBreakpoints from "../../hooks/useBreakpoints";
 import CollectionCard from "../cards/collectionCard";
 import CollectionDataType from "../../types/CollectionDataType";
 import { CollectionCardModel } from "../../models/collectionCard";
-import { SwimLanesProps } from "../../types/props/SwimLanesProps";
 
-const SwimLanes = ({ numColumns, lanesWithNumItems }: SwimLanesProps) => {
+const CollectionLanes = ({ numColumns, lanesWithNumItems, isSwimLane }) => {
   const { isLargerThanLargeTablet } = useBreakpoints();
-  const lanes = lanesWithNumItems.map((lane, key) => (
+  const seeMoreLink = (lane, isSwimLane) =>
+    isSwimLane
+      ? `${DC_URL}/collections/lane/${lane.slug}`
+      : `${DC_URL}/divisions/${lane.slug}`;
```

Comment:
I think this will work better if the "see more" link is passed as a prop instead of `isSwimLane`:

```
<CollectionLanes
  numColumns={numColumns}
  lanesWithNumItems={data?.data?.divisions}
  seeMoreLink={`${DC_URL}/divisions/${lane.slug}`}
/>
```

- `CollectionLanes` does not need to know what the link it should render is,
- the logic is outside of the component,
- it makes it more generic. If there's a new "see more" link in the future, you'd have to update the logic to render another URL. (This may not happen in this case in DC Facelift but consider this as a general pattern to simplify components).